modules/ROOT/nav.adoc

```diff
@@ -60,7 +60,7 @@
 * xref:troubleshooting.adoc[Troubleshooting]
```

Comment:
INSTRUCTIONS FOR EDIT

Revised topic, added icons to address bugs, bring in alignment with current UI. Also changed title and bumped out one level in nav.